### ISSUE TITLE
Show a breakdown of the total size

### ISSUE THREAD

**Issue by sprite-1** [elementary/sideload#76]
## Prerequisites
- [x] I have searched open and closed issues for duplicates.

## Feature
**Is your feature request related to a problem? Please describe.**

As an end-user, it seems absurd to install Telegram which I know is a relatively small application and be greeted by this size with no further information on why it's going to take up this much space.
![Screenshot from 2019-11-02 20-13-40](https://user-images.githubusercontent.com/33685433/68070748-52183b00-fdad-11e9-89a6-c475f16093a1.png)

**Describe the solution you'd like**

While I myself understand that Flatpak requires some common runtimes for the apps to work properly and it's generally a one-time install for the most part, the common layperson that elementary OS aims to target may not. So showing a list of _to-download_ components with a break down of size would be a whole lot better. Maybe even indicating that certain components are runtimes that only need to be downloaded once and not again for subsequent applications. In the name of keeping the UI clutter-free, this can be hidden in a toggleable triangle. (I'll update this later with a mock design)

**Existing work**

I don't know if this exists elsewhere but it would sure help in making elementary OS feel more premium and polished.

**Describe alternatives you've considered**

I haven't considered any other alternatives as I believe this would be the optimal approach. It keeps things simple while still giving the users the information in case they want it.

**Additional context**

None

**Comment by sprite-1** [comment]
Added the mock design. I meant something like that

**Comment by cassidyjames** [comment]
Could we go a step further and subtract the size of the the installed components from the estimate?

**Comment by sprite-1** [comment]
> Could we go a step further and subtract the size of the the installed components from the estimate?

I wasn't sure how Sideload worked behind the scenes but I thought if the required component is already installed, it would automatically be omitted from the total size (thus, also the list)

**Comment by davidmhewitt** [comment]
Yes, that should already be the case. Only components that are to be installed are included in the download size calculation. However, the actual download size may be less as the calculation can't work out any delta update reductions.
